README.md

```diff
@@ -74,59 +67,27 @@ Amongst other things, this will create an admin user for you:
 - The admin interface is at `admin.lvh.me:3000`
 - Access code docs through your local filesystem and update them with `bin/rails yard`
 
-## Testing
+## Testing, linting and formatting
 
-PlaceCal tests are written in minitest.
+PlaceCal tests are written in minitest. We use Rubocop to lint our Ruby code.
 
-Before running the tests please make sure your development environment is up to date (you can run `bin/update` to quickly do that).
+We use [Prettier](https://prettier.io/) to format everything it's able to parse. We run this automatically as part of a pre-commit hook.
 
-You can run the tests with:
+You can run the tests & rubocop with:
 
 ```sh
-make              # will run all unit and system tests then lint check all code
-rails test        # will run all unit tests
-rails test:system # will run system tests
+make test
 ```
 
-Note that the system tests can take a while to run and are quite resource-intensive. To perform more advanced usage like executing only a specific test or test file, see the [Rails documentation on testing](https://guides.rubyonrails.org/testing.html).
-
 ## Documentation for Developers
 
-The documentation for PlaceCal currently stored in notion and can be read [here](https://www.notion.so/gfsc/PlaceCal-developer-handbook-01649b69009340e3ae3035e9cf346f27). There is also a small amount of documentation sprinkled throughout the code itself and can be turned into HTML by running `rails yard`. If you are working with the code and are completely lost you can also try the GFSC discord server where you can prod a human for answers. Good Luck!
+The documentation for PlaceCal currently stored in notion and can be read [here](https://www.notion.so/gfsc/PlaceCal-developer-handbook-01649b69009340e3ae3035e9cf346f27). There is also a small amount of documentation sprinkled throughout the code itself and can be turned into HTML by running `rails yard`. If you are working with the code and are completely lost you can also try the [GFSC discord server](http://discord.gfsc.studio) where you can prod a human for answers. Good Luck!
 
 ### Generating new components
```

Comment:
Should we add a note here about the legacy mountain_view system? I think this could be really confusing for new people otherwise but understand if you want to keep the README ultra simple.

Reply:
sure, have added a line